k8s/lb/ingress.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: extensions/v1beta1
```

Comment:
Ditto: comments. What's Ingress? (Again, just ignorance on my part, but short explanation and a link or two would help.)